src/modules/transaction/utils.ts

```diff
@@ -80,13 +82,13 @@ export function getEtherscanHref(
   return `${getEtherscanOrigin(network)}${pathname}`
 }
 
-export function getEtherscanOrigin(network: number = 1) {
-  switch (network) {
-    case 3:
+export function getEtherscanOrigin(chainId: number = ChainId.ETHEREUM_MAINNET) {
+  switch (chainId) {
```

Comment:
Maybe we need to change the name of the name of the method `getEtherscanHref ` to `getTransactionLink` and `getEtherscanOrigin` to `getTransactionLinkOrigin` by adding


`https://explorer-mumbai.maticvigil.com` and `https://explorer-mainnet.maticvigil.com` for Matic

Reply:
hm you're right. We should change `EtherscanLink` too I think. I'll address both